### PR TITLE
Put some useful information in the README

### LINKED ISSUE #8: Update README.md with useful information
* A summary of the plugin's purpose
* An example `.drone.yml` stanza 
  * If #7 hasn't been done, the `image` drone setting will need to be a placeholder
* Some notes for contributors might be nice:
  * Preferred workflow, code of conduct, etc. (if we have such things)
  * How to run the tests
  * Link to [how to install Helm locally](https://github.com/helm/helm#install)
  * [How to make a local docker registry](https://docs.docker.com/registry/deploying/)
  * [How to use `drone exec` to build and deploy a plugin image](https://docs.drone.io/cli/commands/drone-exec/)

### PR DESCRIPTION
**Pull request by ErinCall**
Fixes #8.

### CODE REVIEW

**Comment by ErinCall** [comment]
This is still underway, but I went ahead and added reviewers in case someone wants to weigh in right away.

**Comment by ErinCall** [comment]
Ok, I think this is as ready as I can make it. I'm going to skip the "information for contributors" part; it's better to do it in #12.

File: docs/parameter_reference.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Parameter reference
+
+## Something here about how to specify non-string types
```
Comment (ErinCall) [inline docs/parameter_reference.md]: Uh I seem to have left some notes-to-self in here

File: docs/parameter_reference.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+| kubernetes_token       | string   | yes      | Token for authenticating to Kubernetes. |
+| service_account        | string   |          | Service account for authenticating to Kubernetes. Default is `helm`. |
+| kubernetes_certificate | string   |          | Base64 encoded TLS certificate used by the Kubernetes cluster's certificate authority. |
+| dry_run                | boolean  |          | Pass `--dry-run` to `helm uninstall`. |
+| timeout                | duration |          | Timeout for any *individual* Kubernetes operation. The uninstallation's full runtime may exceed this duration. |
+| skip_tls_verify        | boolean  |          | Connect to the Kubernetes cluster without checking for a valid TLS certificate. Not recommended in production. |
+
+### Where to put settings
+
+Any setting (with the exception of `prefix`; [see below](#user-content-using-the-prefix-setting)), can go in either the `settings` or `environment` section.
```
Comment (ErinCall) [inline docs/parameter_reference.md]: It seems like this needs more information, like why you'd want to put something in one stanza or the other, but I don't really know enough about drone to tell someone what they should do. Any suggestions?

Comment (josmo) [reply docs/parameter_reference.md]: Lots of this documentation will end up in the http://plugins.drone.io/ (we'll to get a PR in there) so I wouldn't worry too much until it gets used more and we can have more info on what belongs where :)

**Pull request merged**